packages/catalog-search/src/tests/SearchHeader.test.jsx

```diff
@@ -18,24 +19,24 @@ jest.mock('../SearchFilters', () => ({
 }));
 
 describe('SearchHeader', () => {
-  it('displays a SearchBox', () => {
-    render(<MemoryRouter><SearchData><SearchHeader /></SearchData></MemoryRouter>);
+  test('displays a SearchBox', () => {
```

Comment:
Why are you changing `it` to `test` here?

Reply:
No reason in particular, to be honest. We seem to be using `test` more so than `it` lately so I swapped here for that reason. I can swap back to `it` or keep them as `test`. Do you have a strong opinion here?

Reply:
Not really. I always use it because it encourages me to write the test name like a sentence, but I don't actually care much.